install.py

```diff
@@ -226,20 +218,15 @@ def validate_path(path):
     cuda_dir = validate_path(cuda_dir)
     nccl_dir = validate_path(nccl_dir)
     tblis_dir = validate_path(tblis_dir)
-    legate_dir = validate_path(legate_dir)
     thrust_dir = validate_path(thrust_dir)
     curand_dir = validate_path(curand_dir)
     gasnet_dir = validate_path(gasnet_dir)
     cutensor_dir = validate_path(cutensor_dir)
     openblas_dir = validate_path(openblas_dir)
 
-    if legate_dir is None:
-        try:
-            import legate.install_info as lg_install_info
+    import legate.install_info as lg_install_info
 
-            legate_dir = dirname(lg_install_info.libpath)
-        except Exception:
-            pass
+    legate_dir = dirname(lg_install_info.libpath)
```

Comment:
Can we still do the try block, just to give an informative error message if the user hasn't installed legate.core?

Reply:
https://github.com/nv-legate/cunumeric/pull/656/commits/16facef7e98b65e1e0d32f522b0bb8a9acf2a40c